docs/source/github.rst

```diff
@@ -3,6 +3,17 @@ GitHub
 
 How to collaborate to a repository such as `EasyHPC <https://github.com/pescap/EasyHPC>`_? If you want to send Pull Requests to open source directories, you have to follow these steps:
 
+Create an issue
+---------------
+This is very helpful to continue suggesting changes or enhancementes to the code.
```

Comment:
Change to:

To propose changes or enhancements to the code, it is preferable to use the Issues section.